tutorials/fileToJdbc/src/test/java/com/example/FileToJdbcApp/ApplicationTest.java

```diff
@@ -0,0 +1,49 @@
+package com.example.FileToJdbcApp;
+
+import com.datatorrent.api.LocalMode;
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import javax.validation.ConstraintViolationException;
+import java.io.IOException;
+
+/**
+ * Test the DAG declaration in local mode.<br>
+ * The assumption to run this test case is that test_jdbc_table
+ * and meta-table are created already.
```

Comment:
Can we provide a .sql script file as is done in the jdbcIngest and jdbcToJdbc examples ?